src/components/KCheckbox/KCheckbox.vue

```diff
@@ -121,45 +143,139 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+/* Component mixins */
+
+@mixin kCheckboxIcon {
+  inset: 0;
+  left: calc(50% - 2.4px); // 2px is not enough, 3px is too much...
+  pointer-events: none;
+  position: absolute;
+  top: calc(50% + 1.75px); // 1px is not enough, 2px is too much...
+  transform: translate(-50%, -50%);
+  z-index: 1;
+}
 
-@import '@/styles/mixins';
+/* Component styles */
 
 .k-checkbox {
-  .k-input {
-    @include input-type-checkbox;
+  align-items: center;
+  display: inline-flex;
+
+  &.has-description {
+    align-items: flex-start;
+
+    .checkbox-input-wrapper {
+      margin-top: 3px; // align with label
+    }
+  }
+
+  .checkbox-input-wrapper {
+    display: flex;
+    position: relative;
+  }
+
+  /* Checkbox styles */
+  .checkbox-input {
+    @include radioCheckboxDefaults;
+
+    border-radius: var(--kui-border-radius-20, $kui-border-radius-20);
```

Comment:
I'm assuming we're overriding the radius of the mixin above? If yes, can you please add a comment on the line above?

Reply:
No - the mixin doesn't have the `border-radius` property (because it's used in both `KRadio` and `KCheckbox` - there's no default value we want for it). So here we're literally setting `border-radius` property (same in `KRadio`). I will add a comment mentioning that.